app/main.ipynb

```diff
@@ -216,15 +265,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "column_means = A.mean(axis=1)\n",
+    "print(column_means)"
```

Comment:
The task requires calculating the standard deviation by the y-axis (axis=1), but the code currently calculates the mean. You should use `np.std()` instead of `np.mean()` to get the standard deviation.